src/tracely/exceptions/error_messages.py

```diff
@@ -43,7 +43,8 @@ class ValidationErrorMessage:
     INVALID_PING_BATCH_SIZE_FOR_MAP_MATCHING = "ping_batch_size cannot be less than 2"
     INVALID_MIN_SIZE_FOR_STOP_EVENTS_DETECTION = "min_size cannot be less than 2"
     INVALID_MAX_DIST_RATIO_FOR_IMPUTATION = "max_dist_ratio can not be less than 0, but got {}"
-
+    INVALID_TRACE_FOR_OVERLAP_ESTIMATION = "Input traces must contain latitude, longitude and timestamp for all pings."
```

Comment:
but you have allowed None for latitude, longitude right? isn't this a very generic error message?